src/PlotlyEditorWithPlot.js

```diff
@@ -0,0 +1,58 @@
+import React, {Component} from 'react';
+import createPlotComponent from 'react-plotly.js/factory';
+import PlotlyEditor from '../src';
+import PropTypes from 'prop-types';
+
+class PlotlyEditorWithPlot extends Component {
+  constructor(props) {
+    super();
+    this.state = {graphDiv: {}};
```

Comment:
The new component only stores the graphDiv in state, which it gets from Plot.

So the lifecycle of a user edit is now:
* `data`/`layout` from `PlotlyEditor` up to `PlotlyEditorWithPlot` up to `App`
* `setState` causes `render` for `App`
*  `data`/`layout` from `App` down to `PlotlyEditorWithPlot` down to `Plot`
* `graphDiv` from `Plot` up to `PlotlyEditorWithPlot`
* `setState` causes `render` for `PlotlyEditorWithPlot`
* `graphDiv` from `PlotlyEditorWithPlot` down to `PlotlyEditor`